### ISSUE TITLE
Stats page not updating with pool information.

### ISSUE THREAD

**Issue by aznjack31** [c00w/bitHopper#697]
I pointed my miner to the bithopper address and I can see the mining in all the different pools but I the stats won't update on http://ec2-54-242-233-147.compute-1.amazonaws.com:8339/stats#

Why is that?

**Comment by c00w** [comment]
Whats not updating about them. The white shares count on the right? Or one of the undefineds?

**Comment by c00w** [comment]
Oh I see, the actual shares. And the getworks/rejects information is broken.

**Comment by c00w** [comment]
This admin page is still a work in progress. I just updated the share counts on the left for the pools so the percentage shows up correctly.

**Comment by aznjack31** [comment]
Yeah the actual shares and miners don't show up or update on the admin page. All I see is the mh changing. Is there an older working version I could try instead so I know how much each user is mining? Thanks for the quick response!

**Comment by aznjack31** [comment]
Internal server error now lol. looks like I got to restart everything again.

**Comment by c00w** [comment]
What error occured?

**Comment by c00w** [comment]
Version v0.2.6.7 or v0.2.6.6 probably will work best if you want the old style admin interface.
